vm/core.go

```diff
@@ -353,6 +357,7 @@ func (i *Instance) Run() (err error) {
 			}
 		}
 		i.insCount++
+		time.Sleep(i.clockPeriod)
```

Comment:
This is a systematic function call that will adversely impact performance for all users. Please make it conditional on `i.clockPeriod > 0`.